### ISSUE TITLE
ONYX-989: Internal Error when trying to display a user comments with data from previous versions of Onyx

### ISSUE THREAD

**Issue by ymarcon** [obiba/onyx#993]
Jira issue originally created by user @dwainelson:

Error from 1.5.4

2009-11-01 11:19:42,363 - INFO  - tate.StageExecutionContext - castEvent(complete) from stage 'ConclusionQuestionnaire' in state 'QuartzInProgressState'
2009-11-01 11:19:51,869 - ERROR - apache.wicket.RequestCycle - Exception in rendering component: [Component id = action]
org.apache.wicket.WicketRuntimeException: Exception in rendering component: [Component id = action]
        at org.apache.wicket.Component.renderComponent(Component.java:2519)
        at org.apache.wicket.markup.html.WebComponent.onRender(WebComponent.java:60)
        at org.apache.wicket.Component.render(Component.java:2317)
        at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
        at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
        at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
        at org.apache.wicket.Component.renderComponent(Component.java:2480)
        at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
        at org.apache.wicket.Component.render(Component.java:2317)
        at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
        at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
        at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
        at org.apache.wicket.markup.html.panel.Fragment.renderFragment(Fragment.java:278)
        at org.apache.wicket.markup.html.panel.Fragment.onComponentTagBody(Fragment.java:206)
        at org.apache.wicket.Component.renderComponent(Component.java:2480)
        at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
        at org.apache.wicket.Component.render(Component.java:2317)
        at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
        at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
        at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
        at org.apache.wicket.Component.renderComponent(Component.java:2480)
        at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
        at org.apache.wicket.Component.render(Component.java:2317)
        at org.apache.wicket.markup.html.list.Loop.renderItem(Loop.java:201)
        at org.apache.wicket.markup.html.list.Loop.renderChild(Loop.java:190)
        at org.apache.wicket.markup.repeater.AbstractRepeater.onRender(AbstractRepeater.java:100)
        at org.apache.wicket.Component.render(Component.java:2317)
        at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
        at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
        at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:639)
        at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:112)
        at org.apache.wicket.Component.renderComponent(Component.java:2480)
        at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
        at org.apache.wicket.Component.render(Component.java:2317)
        at org.apache.wicket.Component.renderComponent(Component.java:2420)
        at org.apache.wicket.ajax.AjaxRequestTarget.respondComponent(AjaxRequestTarget.java:785)
        at org.apache.wicket.ajax.AjaxRequestTarget.respondComponents(AjaxRequestTarget.java:662)
        at org.apache.wicket.ajax.AjaxRequestTarget.respond(AjaxRequestTarget.java:577)
        at org.apache.wicket.request.AbstractRequestCycleProcessor.respond(AbstractRequestCycleProcessor.java:104)
        at org.apache.wicket.RequestCycle.processEventsAndRespond(RequestCycle.java:1181)
        at org.apache.wicket.RequestCycle.step(RequestCycle.java:1252)
        at org.apache.wicket.RequestCycle.steps(RequestCycle.java:1353)
        at org.apache.wicket.RequestCycle.request(RequestCycle.java:493)
        at org.apache.wicket.protocol.http.WicketFilter.doGet(WicketFilter.java:355)
        at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:200)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.springframework.orm.hibernate3.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:198)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
        at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
        at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
        at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
        at java.lang.Thread.run(Thread.java:636)
Caused by: org.springframework.context.NoSuchMessageException: No message found under code '' for locale 'en'.
        at org.springframework.context.support.AbstractMessageSource.getMessage(AbstractMessageSource.java:161)
        at org.springframework.context.support.DelegatingMessageSource.getMessage(DelegatingMessageSource.java:71)
        at org.springframework.context.support.AbstractApplicationContext.getMessage(AbstractApplicationContext.java:987)
        at org.obiba.wicket.model.MessageSourceResolvableStringModel.getObject(MessageSourceResolvableStringModel.java:70)
        at org.apache.wicket.Component.getModelObject(Component.java:1558)
        at org.apache.wicket.Component.getModelObjectAsString(Component.java:1580)
        at org.apache.wicket.markup.html.basic.Label.onComponentTagBody(Label.java:111)
        at org.apache.wicket.Component.renderComponent(Component.java:2480)
        ... 60 more
2009-11-01 11:30:13,682 - INFO  - $MenuItemSelectionBehavior - linkPageClass = HomePage, pageClass = HomePage
2009-11-01 11:30:13,682 - INFO  - $MenuItemSelectionBehavior - linkPageClass = ParticipantSearchPage, pageClass = HomePage
2009-11-01 11:30:15,814 - INFO  - $MenuItemSelectionBehavior - linkPageClass = HomePage, pageClass = ParticipantSearchPage

*********************************************
Error from 1.5-SNAPSHOT

2009-11-01 11:50:13,013 - INFO  - resource.PropertiesFactory - Loading properties files from file:/var/lib/tomcat6/webapps/ohs-webapp/WEB-INF/classes/org/obiba/onyx/webapp/participant/panel/UnlockInterviewPanel.properties
2009-11-01 11:50:22,880 - ERROR - apache.wicket.RequestCycle - Error attaching this container for rendering: [MarkupContainer [Component id = content]]
org.apache.wicket.WicketRuntimeException: Error attaching this container for rendering: [MarkupContainer [Component id = content]]
        at org.apache.wicket.MarkupContainer.onBeforeRenderChildren(MarkupContainer.java:1601)
        at org.apache.wicket.Component.onBeforeRender(Component.java:3682)
        at org.apache.wicket.Component.internalBeforeRender(Component.java:1002)
        at org.apache.wicket.Component.beforeRender(Component.java:1034)
        at org.apache.wicket.MarkupContainer.onBeforeRenderChildren(MarkupContainer.java:1589)
        at org.apache.wicket.Component.onBeforeRender(Component.java:3682)
        at org.apache.wicket.Component.internalBeforeRender(Component.java:1002)
        at org.apache.wicket.Component.beforeRender(Component.java:1034)
        at org.apache.wicket.MarkupContainer.onBeforeRenderChildren(MarkupContainer.java:1589)
        at org.apache.wicket.Component.onBeforeRender(Component.java:3682)
        at org.apache.wicket.Component.internalBeforeRender(Component.java:1002)
        at org.apache.wicket.Component.beforeRender(Component.java:1034)
        at org.apache.wicket.MarkupContainer.onBeforeRenderChildren(MarkupContainer.java:1589)
        at org.apache.wicket.Component.onBeforeRender(Component.java:3682)
        at org.apache.wicket.extensions.ajax.markup.html.modal.ModalWindow.onBeforeRender(ModalWindow.java:740)
        at org.apache.wicket.Component.internalBeforeRender(Component.java:1002)
        at org.apache.wicket.Component.beforeRender(Component.java:1034)
        at org.apache.wicket.Component.prepareForRender(Component.java:2160)
        at org.apache.wicket.ajax.AjaxRequestTarget.respondComponent(AjaxRequestTarget.java:762)
        at org.apache.wicket.ajax.AjaxRequestTarget.respondComponents(AjaxRequestTarget.java:662)
        at org.apache.wicket.ajax.AjaxRequestTarget.respond(AjaxRequestTarget.java:577)
        at org.apache.wicket.request.AbstractRequestCycleProcessor.respond(AbstractRequestCycleProcessor.java:104)
        at org.apache.wicket.RequestCycle.processEventsAndRespond(RequestCycle.java:1181)
        at org.apache.wicket.RequestCycle.step(RequestCycle.java:1252)
        at org.apache.wicket.RequestCycle.steps(RequestCycle.java:1353)
        at org.apache.wicket.RequestCycle.request(RequestCycle.java:493)
        at org.apache.wicket.protocol.http.WicketFilter.doGet(WicketFilter.java:355)
        at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:200)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.springframework.orm.hibernate3.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:198)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
        at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
        at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
        at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
        at java.lang.Thread.run(Thread.java:636)
Caused by: java.lang.NullPointerException
        at org.obiba.onyx.engine.ActionDefinition.getLabel(ActionDefinition.java:99)
        at org.obiba.onyx.webapp.action.panel.InterviewLogPanel.getActionModel(InterviewLogPanel.java:183)
        at org.obiba.onyx.webapp.action.panel.InterviewLogPanel.addCommonFragmentComponents(InterviewLogPanel.java:162)
        at org.obiba.onyx.webapp.action.panel.InterviewLogPanel.access$100(InterviewLogPanel.java:54)
        at org.obiba.onyx.webapp.action.panel.InterviewLogPanel$CommentFragment.<init>(InterviewLogPanel.java:129)
        at org.obiba.onyx.webapp.action.panel.InterviewLogPanel$1.populateItem(InterviewLogPanel.java:99)
        at org.apache.wicket.markup.html.list.Loop.onPopulate(Loop.java:135)
        at org.apache.wicket.markup.repeater.AbstractRepeater.onBeforeRender(AbstractRepeater.java:127)
        at org.apache.wicket.Component.internalBeforeRender(Component.java:1002)
        at org.apache.wicket.Component.beforeRender(Component.java:1034)
        at org.apache.wicket.MarkupContainer.onBeforeRenderChildren(MarkupContainer.java:1589)
        ... 43 more
2009-11-01 11:50:25,196 - INFO  - $MenuItemSelectionBehavior - linkPageClass = HomePage, pageClass = HomePage
